packages/commercetools/composables/src/useUser/authenticate.ts

```diff
@@ -0,0 +1,15 @@
+import {CustomerSignMeInDraft, CustomerSignMeUpDraft} from '@vue-storefront/commercetools-api/lib/src/types/GraphQL';
```

Comment:
is there a reason for it to be in different file than factory params?
It's a part of that module

Reply:
yes, I've spoke with @lukeromanowicz about it and it is easier to test it alone and it's cleaner, like in the useCart composable

Reply:
It's argueable if its cleaner but I get the argument with testing